server/game/cards/13-O4B/Fort51.js

```diff
@@ -0,0 +1,66 @@
+const GameActions = require('../../GameActions/index.js');
+const OutfitCard = require('../../outfitcard.js');
+
+class Fort51 extends OutfitCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onGadgetInvented: event => event.gadget.hasKeyword('Weapon') || event.gadget.getType() === 'dude' && event.player === this.controller
+            },
+            handler: context => {
+                this.game.promptForSelect(context.player, {
```

Comment:
This works too, but it does not trigger specific events for selecting other targets that are required for some reacts (or can be in the future). That is why we should use a specific function `selectAnotherTarget`.
```suggestion
                context.ability.selectAnotherTarget(context.player, context, {
```